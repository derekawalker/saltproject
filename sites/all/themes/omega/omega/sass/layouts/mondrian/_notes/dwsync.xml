<?xml version="1.0" encoding="utf-8" ?><dwsync><file name="mondrian.layout.scss" server="ftp.beheros.com//public_html/" local="3547078419" remote="3520303200" Dst="-1" /></dwsync>