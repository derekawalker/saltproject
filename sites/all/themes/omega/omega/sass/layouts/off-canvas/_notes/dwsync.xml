<?xml version="1.0" encoding="utf-8" ?><dwsync><file name="_off-canvas.scss" server="ftp.beheros.com//public_html/" local="3547078419" remote="3520303200" Dst="-1" /><file name="off-canvas.layout.scss" server="ftp.beheros.com//public_html/" local="3547078419" remote="3520303200" Dst="-1" /></dwsync>