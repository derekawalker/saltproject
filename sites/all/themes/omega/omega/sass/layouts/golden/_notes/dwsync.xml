<?xml version="1.0" encoding="utf-8" ?><dwsync><file name="golden.layout.scss" server="ftp.beheros.com//public_html/" local="3547078418" remote="3520303200" Dst="-1" /></dwsync>